src/Pages/Login/Login.js

```diff
@@ -1,12 +1,51 @@
-import React, { Component } from "react";
+import React from "react";
+import "./Login.scss";
+import "../../Styles/reset.scss";
 
-class Login extends Component {
+class Login extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      checkbox: false,
+    };
   }
+  IsChecked = () => {
+    this.setState({ checkbox: !this.state.checkbox });
+    console.log(this.state.checkbox);
+  };
+
   render() {
-    return <></>;
+    return (
+      <div className="Login">
+        <div className="section-login">
+          <p className="login-title">로그인</p>
```

Comment:
우리는 이제 sass의 nesting 기능을 이용합니다.
그래서 className을 작성하실 때, 굳이 부모 태그의 className을 작성하면서 태그 하나하나 이름을 분리해 주지 않아도,nesting 기능을 통해 css 속성이 분리될 수 있습니다. 그래서 위와 같은 태그의 경우도 아래와 같이 간단하게 className을 작성해주실 수 있습니다.

고로 아래와 같이 작성해주시면 되겠죠?!
```suggestion
          <p className="title">로그인</p>
```

다른 className들도 간단하게 변경해보세요!
더불어 그런 의미에서 react에서는 BEM이나 snake case보다는 camelCase 네이밍이 많이 쓰입니다.

Reply:
넵 :)! 저희팀에서 초반에 클래스명을 snake case 로 하기로 해서 일부로 다같이 이렇게 작성했던 것인데,
멘토님 리뷰 보고 앞으로는 camelCase 로 작성하기로 하였습니다 :)
감사합니다 !😊